File: back/src/main/java/com/myweather/api/models/weather/data/Forecast.java
package com.myweather.api.models.weather.data;


/**
 * Created by javierfz on 7/20/17.
 */
public class Forecast {

   /**
    * Internal properties
    */
   private String day;
   private String date;
   private int low;
   private int high;
   private String text;
   private int code;

   /**
    * setters
    */
   public void setDay(String day) {
      this.day = day;
   }

   public void setDate(String date) {
      this.date = date;
   }

   public void setLow(int low) {
      this.low = low;
   }

   public void setHigh(int high) {
      this.high = high;
   }

   public void setText(String text) {
      this.text = text;
   }

   public void setCode(int code) {
      this.code = code;
   }

   /**
    * getters
    */
   public String getDay() {
      return day;
   }

   public String getDate() {
      return date;
   }

   public int getLow() {
      return low;
   }

   public int getHigh() {
      return high;
   }

   public String getText() {
      return text;
   }

   public int getCode() {
      return code;
   }

   /**
    * Constructor
    */
   public Forecast(){}


   @Override
   public String toString() {
      return "Forecast{" +
            ", day='" + day + '\'' +
            ", date='" + date + '\'' +
            ", low='" + low + '\'' +
            ", high='" + high + '\'' +
            ", text='" + text + '\'' +
            ", code='" + code +
            '}';
   }

}

File: front-reactive/src/app/commons/models/data/weatherData/forecast.ts
export class Forecast {

  private _day: string
  private _date: string;
  private _low: string;
  private _high: string;
  private _text: string;
  private _code: string;

  set day(value: string) {
    this._day = value;
  }

  set date(value: string) {
    this._date = value;
  }

  set low(value: string) {
    this._low = value;
  }

  set high(value: string) {
    this._high = value;
  }

  set text(value: string) {
    this._text = value;
  }

  set code(value: string) {
    this._code = value;
  }
  get day(): string {
    return this._day;
  }

  get date(): string {
    return this._date;
  }

  get low(): string {
    return this._low;
  }

  get high(): string {
    return this._high;
  }

  get text(): string {
    return this._text;
  }

  get code(): string {
    return this._code;
  }


}
